### PR TITLE
Generic idle state

### PR DESCRIPTION
**Pull request by Jeff-Moreau**
Test state for everyone to see how to call and use animations

### CODE REVIEW

File: Source/SteveSloth/Private/MyEnemyManager.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/****************************************************************************************
+* Copyright:
+* Name:
+* Script: MyEnemyManager.cpp
+* Date: 
+* Description: 
+* TODO: 
+* Known Bugs: 
+*****************************************************************************************/
```
Comment (Saretzky) [inline Source/SteveSloth/Private/MyEnemyManager.cpp:9]: fill this out

File: Source/SteveSloth/Public/MyEnemyManager.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/****************************************************************************************
+* Copyright:
+* Name:
+* Script: MyEnemyManager.h
+* Date: 
+* Description: 
+* TODO: 
+* Known Bugs: 
+*****************************************************************************************/
```
Comment (Saretzky) [inline Source/SteveSloth/Public/MyEnemyManager.h:9]: fill this out

File: Source/SteveSloth/Private/MyEnemyBaseClass.cpp [hunk 3]
```diff
@@ -9,17 +9,22 @@
  ****************************************************************************************/
 
 #include "MyEnemyBaseClass.h"
+#include "MyGenericEnemyIdleState.h"
```
Comment (Saretzky) [inline Source/SteveSloth/Private/MyEnemyBaseClass.cpp:12]: this should be in the header file

**Review by Saretzky — CHANGES_REQUESTED** [review]

File: Source/SteveSloth/Private/MyEnemyBaseClass.cpp (hunk 3 above)
Comment (Jeff-Moreau) [reply Source/SteveSloth/Private/MyEnemyBaseClass.cpp:12]: This needs to stay here do to circular dependency. and with the include in the cpp file all im doing is grabbing a reference to it not actually using it.. In the header file i would have access to everything and i dont need that, also it causes circular dependency.

File: Source/SteveSloth/Public/MyEnemyManager.h (hunk 2 above)
Comment (Jeff-Moreau) [reply Source/SteveSloth/Public/MyEnemyManager.h:9]: Sorry I have not even got into the enemymanager yet.. that was something richard and i were talking about and I created it then dropped it.. I will be picking it back up before the end of class..

File: Source/SteveSloth/Private/MyEnemyBaseClass.cpp (hunk 3 above)
Comment (Saretzky) [reply Source/SteveSloth/Private/MyEnemyBaseClass.cpp:12]: sounds good

File: Source/SteveSloth/Public/MyEnemyManager.h (hunk 2 above)
Comment (Saretzky) [reply Source/SteveSloth/Public/MyEnemyManager.h:9]: I still would say just fill out the top with basic info

Comment (Saretzky) [reply Source/SteveSloth/Public/MyEnemyManager.h:9]: the actual script can stay empty for now

Comment (Jeff-Moreau) [reply Source/SteveSloth/Public/MyEnemyManager.h]: All done

File: Source/SteveSloth/Private/MyEnemyManager.cpp (hunk 1 above)
Comment (Jeff-Moreau) [reply Source/SteveSloth/Private/MyEnemyManager.cpp]: Done

**Review by Saretzky — CHANGES_REQUESTED** [review]
looks good

**Comment by Saretzky** [comment]
> looks good

oops

**Review by Saretzky — APPROVED** [review]
looks good

**Comment by Jeff-Moreau** [comment]
> > looks good
> 
> oops

That wasnt very nice

**Pull request merged**